src/cli.ts

```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env node
 
 import { Command } from 'commander';
-import { TypeScriptGenerator } from './generators/typescript';
 import { CSharpGenerator } from './generators/csharp';
+import { PythonGenerator } from './generators/python';
+import { TypeScriptGenerator } from './generators/typescript';
 import { SchemaLoader } from './loader';
 import path from 'path';
 import fs from 'fs';
 
-
 const generators = {
   typescript: TypeScriptGenerator,
-  csharp: CSharpGenerator
+  csharp: CSharpGenerator,
+  python: PythonGenerator
```

Comment:
In context of multiple python backends in future, maybe we need to use something like `fhir-py` or `python-fhir-py`?

Also (some days) we can add `description` to the right side to improve --help message.